frontend/demo/src/api/airyConfig.ts

```diff
@@ -1,15 +1,16 @@
 import {getAuthToken} from './webStore';
 
 export class AiryConfig {
-  static API_URL = 'airy.api';
+  static API_URL = 'http://api.airy';
   static NODE_ENV = process.env.NODE_ENV;
+  static FACEBOOK_APP_ID = 'CHANGE_ME';
```

Comment:
We need to move this to a config file.

Reply:
Late to the discussion, but because this is served by an nginx pod, which has access to the env, we could also inject these values using lua scripts https://github.com/bungle/lua-resty-template cc @lucapette and @AitorAlgorta

Reply:
Yeah, could be one possibility. Let's discuss this in [this](https://github.com/airyhq/airy/issues/421) ticket.